hera_qm/xrfi.py

```diff
@@ -557,6 +556,21 @@ def waterfall2flags(waterfall, uv):
     return flag_array
 
 
+def normalize_wf(wf, wfp):
+    """ Normalize waterfall to account for data already flagged.
+    Args:
+        wf -- Waterfall of fractional flags.
+        wfp -- Waterfall of prior fractional flags. Size must match wf.
+    Returns:
+        wf_norm -- Waterfall of fractional flags which were not flagged prior.
+                   Note if wfp[i, j] == 1, then wf_norm[i, j] will be NaN.
+    """
+    if wf.shape != wfp.shape:
+        raise AssertionError('waterfall and prior waterfall must be same shape.')
+    wf_norm = (wf - wfp) / (np.ones_like(wf) - wfp)
```

Comment:
See my main review comment. I think that we can be more clever here with a `where` statement, and have the output have a value of `1` instead  of `NaN` when `wfp == 1`.

Reply:
I agree this is probably not the best solution. However, if possible I would like to carry the fact that `wf_norm` didn't actually know anything about these pixels. The idea is that if it was previously flagged, and we map that to a 1, it will mess up the thresholding done on lines 591 and 592. What would you think of this compromise:
`wf_norm = np.where(wfp < 1, (wf - wfp) / (np.ones_like(wf) - wfp), np.nan)`
This gets rid of the warnings, but retains the nans. Is that still poor form?